Spv.Generator/Instruction.cs

```diff
@@ -228,5 +229,18 @@ public bool Equals(Operand obj)
         {
             return obj is Instruction instruction && Equals(instruction);
         }
+
+        public override string ToString()
+        {
+            var operandLabels = new Dictionary<Specification.Op, string[]>
+            {
+                {Specification.Op.OpConstant, new []{"Value"}},
```

Comment:
```suggestion
                { Specification.Op.OpConstant, new [] { "Value" } },
```
Same for the ones below.